.mk/develop.mk

```diff
@@ -42,7 +47,7 @@ run-docker: run-docker-teardown ## run the app under docker-compose
 	@# podman (at least) doesn't seem to like multi-arch images, and sometimes picks the wrong one (e.g. amd64 on arm64)
 	@# We also need to remove the build: directives to use ko builds
 	@# ko resolve will fill in the image: field in the compose file, but it adds a yaml document separator
-	@sed -e '/^  *build:/d'  -e 's|  image: minder:latest|  image: ko://github.com/stacklok/minder/cmd/server|' docker-compose.yaml | KO_DOCKER_REPO=$(KO_DOCKER_REPO) ko resolve --base-import-paths --platform linux/$(DOCKERARCH) -f - | sed 's/^--*$$//' > .resolved-compose.yaml
+	@yq e $(YQ_BUILD_REPLACE_STRING) docker-compose.yaml | KO_DOCKER_REPO=$(KO_DOCKER_REPO) ko resolve --base-import-paths --platform linux/$(DOCKERARCH) -f - | sed 's/^--*$$//' > .resolved-compose.yaml
```

Comment:
Could we do without `yq` or fix the docs (README, docs/docs/developer_guide/get-hacking.md and docs/docs/run_minder_server/run_the_server.md) to include yq installation?

Reply:
> Could we do without yq

We can, but `yq` is becoming pretty standard. Even GH-hosted runners have it installed. 

> fix the docs (README, docs/docs/developer_guide/get-hacking.md and docs/docs/run_minder_server/run_the_server.md) to include yq installation

Yeah, I will add it to docs.

Reply:
another thing might be to put it into the make bootstrap command in `.mk/develop.mk`

Reply:
Done, added docs and to the `bootstrap` command.

Reply:
thank you. It seems that you'd need to install `github.com/mikefarah/yq/v4@latest`, without the latest tag I get:

```bash
go install github.com/mikefarah/yq/v4
no required module provides package github.com/mikefarah/yq/v4; to add it:
        go get github.com/mikefarah/yq/v4
```

Reply:
`cd tools && go mod tidy`? It should work after that